lessons/day_07/python/data.py

```diff
@@ -0,0 +1,42 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from models import Base, Skill, Person
+import json
+
+
+engine = create_engine('sqlite://')
+Session = sessionmaker(bind=engine)
+session = Session()
+
+# Setup of in memory database, from data.json if tables doesn´t exist
+if not engine.dialect.has_table(engine.connect(), "skills"):
+    Base.metadata.create_all(bind=engine)
+    data = json.load(open('../datasource/data.json'))
```

Comment:
same here you can use Path
```suggestion
    data = json.load(open(Path.cwd()/'datasource/data.json'))
```